src/routes/imports/add.ts

```diff
@@ -0,0 +1,50 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+import {
+  ProcessingImport,
+  processingImportSchema,
+  InvalidImport,
+  invalidImportSchema,
+} from '../../models/imports';
+import { B64XLSXFile, b64XLSXFile } from './types';
```

Comment:
What's the added benefit of this schema and type? Why not use [FileToServer](https://github.com/withthegrid/platform-sdk/blob/1791b662a3629a7d626cbb90eb602f8572899fe5/src/models/file-to-server.ts)?

Applies to other occurrences as well

Reply:
None really, I'll replace with `FileFromServer`. 
The only advantage is that it enforces the mime-type. But I could find a way to extend `FileFromServer` to do so too if it's wanted

Reply:
I don't think it is needed. We currently have mimetype optional in uploads by the way, see https://github.com/withthegrid/platform/blob/318093f40c9154ff6e4f30d49fe44ad589ec4b9f/src/services/files/files.ts#L42-L46.